frontend/public/components/routes/_create-route.scss

```diff
@@ -3,3 +3,11 @@
     margin-bottom: 0;
   }
 }
+
+.co-create-route__weight-input {
+  max-width: 100% !important;
```

Comment:
OK, it looks like we have a some styles here causing the narrow width:

https://github.com/openshift/console/blob/master/frontend/public/style/_base.scss#L21-L29

We might look at disabling those. I know I commented on the widths, but I'd remove this rule for now. We can revisit this generally for console.

@rhamilto @sg00dwin fyi